cothink2/client/messagebus.js

```diff
@@ -0,0 +1,44 @@
+/* Simple messagebus/pubsub thingy.
+ */
+
+msgbus = function (jQuery) {
+    channels = {}
```

Comment:
Here you're definitely missing the variable declarations right? Also, why does this object initialization need jQuery?